lms/djangoapps/instructor/views/api.py

```diff
@@ -1733,7 +1734,7 @@ def generate_registration_codes(request, course_id):
         log.exception('Exception at creating pdf file.')
         pdf_file = None
 
-    from_address = microsite.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
+    from_address = theming_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
```

Comment:
@mattdrayer  please your thoughts here as per my some above comments to the same things ?